net/ser2net/files/etc/init.d/ser2net

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh /etc/rc.common
+START=75
+STOP=10
+CONFIGFILE="/tmp/ser2net.conf"
+
+start() {
+	pidof ser2net | awk '{print $2}' > /tmp/ser2net
+	PID=$(cat /tmp/ser2net)
+	rm /tmp/ser2net
+    if [ ! $PID ]; then
+		update_configfile
+        ser2net -c "$CONFIGFILE"
+
+    else
+        echo "Ser2Net: is already running"
+    fi
+}
+
+stop() {
+	pidof ser2net | sed -e "s/$$//g" > /tmp/ser2net_pid
+	PID=`cat /tmp/ser2net_pid`
+	rm -f /tmp/se2net_pid
+	kill -9 $PID 	
+}
```

Comment:
just make this a proper procd service. there's no need for this mucking about trying to track pidfiles manually.  it would be _simpler_ to do this properly.

Reply:
Ok will look into that.

Reply:
Concur with @karlp.